meta-mender-qemu/recipes-bsp/u-boot/patches/0001-vexpress-qemu-specific-fixes-for-Flash.patch

```diff
@@ -1,23 +1,41 @@
-From c0eb625c6cbe1537387fed7210c5c80ba7a1b189 Mon Sep 17 00:00:00 2001
+From 67eb032ce5df5a956d440dc55695739de5bc17d5 Mon Sep 17 00:00:00 2001
 From: Kristian Amlie <kristian.amlie@northern.tech>
 Date: Tue, 20 Feb 2018 13:52:07 +0100
-Subject: [PATCH 1/1] vexpress-qemu specific fixes for Flash.
+Subject: [PATCH] vexpress-qemu specific fixes for Flash.
 
 These are only needed because vexpress-qemu is not originally a Flash
 based board.
 
 Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>
 ---
- configs/vexpress_ca9x4_defconfig  |  2 ++
+ configs/vexpress_ca9x4_defconfig  |  5 ++++-
  include/configs/vexpress_ca9x4.h  | 10 ++++++++++
  include/configs/vexpress_common.h |  3 +--
- 3 files changed, 13 insertions(+), 2 deletions(-)
+ 3 files changed, 15 insertions(+), 3 deletions(-)
 
 diff --git a/configs/vexpress_ca9x4_defconfig b/configs/vexpress_ca9x4_defconfig
-index f10e73f86e..aba00879f3 100644
+index f10e73f86e..73767e719f 100644
 --- a/configs/vexpress_ca9x4_defconfig
 +++ b/configs/vexpress_ca9x4_defconfig
-@@ -35,3 +35,5 @@ CONFIG_BAUDRATE=38400
+@@ -1,7 +1,7 @@
+ CONFIG_ARM=y
+ CONFIG_TARGET_VEXPRESS_CA9X4=y
+ CONFIG_SYS_TEXT_BASE=0x60800000
+-CONFIG_ENV_SIZE=0x40000
++CONFIG_ENV_SIZE=0x20000
+ CONFIG_NR_DRAM_BANKS=2
+ CONFIG_ENV_SECT_SIZE=0x40000
+ CONFIG_DISTRO_DEFAULTS=y
+@@ -24,6 +24,8 @@ CONFIG_ENV_IS_IN_FLASH=y
+ CONFIG_ENV_ADDR=0x47F80000
+ CONFIG_MTD=y
+ CONFIG_MTD_NOR_FLASH=y
++CONFIG_MTDIDS_DEFAULT="nor2=40000000.flash"
++CONFIG_MTDPARTS_DEFAULT="mtdparts=40000000.flash:1m(u-boot)ro,-(ubi)"
```

Comment:
The specific things that you are adding in this patch are the exact things we would like not to be added, so that the auto-patcher will do it for us. It's how we test the auto-patcher. The existing things in this patch are here because they are too difficult to the patcher to catch, or specific to our setup.

Reply:
I'm not sure what you are saying here. Should I not need to add these?  The current Zeus branch build of vexpress-qemu-flash fails with:
```
NOTE: Setscene tasks completed
ERROR: u-boot-fw-utils-1_2019.07-r0 do_configure: U-Boot configuration vexpress_ca9x4_config has setting:
CONFIG_MTDIDS_DEFAULT=""
but Mender expects:
CONFIG_MTDIDS_DEFAULT="nor2=40000000.flash"
Please fix U-Boot's configuration file.
ERROR: u-boot-fw-utils-1_2019.07-r0 do_configure: Execution of '/work/dmoseley/yocto-build/Mender/yocto/zeus/build/tmp/work/vexpress_qemu_flash-poky-linux-gnueabi/u-boot-fw-utils/1_2019.07-r0/temp/run.do_configure.54691' failed with exit code 1:
CONFIG_MTDIDS_DEFAULT=""
WARNING: exit code 1 from a shell command.

ERROR: Logfile of failure stored in: /work/dmoseley/yocto-build/Mender/yocto/zeus/build/tmp/work/vexpress_qemu_flash-poky-linux-gnueabi/u-boot-fw-utils/1_2019.07-r0/temp/log.do_configure.54691
ERROR: Task (/work/dmoseley/community/yocto/zeus/src/poky/meta/recipes-bsp/u-boot/u-boot-fw-utils_2019.07.bb:do_configure) failed with exit code '1'
```

Reply:
NM.  It turns out that [this line](https://github.com/mendersoftware/meta-mender/blob/64baa4815e03b89a100976c6e171733a579f1eab/meta-mender-raspberrypi/recipes-bsp/u-boot/u-boot-raspberrypi.inc#L7) was disabling autoconfig for _all_ targets rather than just RPI.  I've got it fixed locally and will push a new version soon.